### ISSUE TITLE
Upgrade 2.1.5 to 2.2: Upgrading fmDNS Schema 	failed

### ISSUE THREAD

**Issue by internethering** [WillyXJ/facileManager#313]
to debug I switched on general_log of mariadb:

1. problem: after failed once It will fail everytime at the to ALTER TABLE-Statements:
```
ALTER TABLE `fm_dns_acls` ADD `acl_parent_id` INT NOT NULL DEFAULT '0' AFTER `server_serial_no`

#1060 - Duplicate column name 'acl_parent_id'
```

I commented out the two ALTER TABLE statements, the error:

```
INSERT INTO `fm_dns_acls` (account_id, server_serial_no, acl_parent_id, acl_addresses, acl_status)
						VALUES (1, 0, 2, '10/14', 'active')

#1364 - Field 'acl_name' doesn't have a default value
```

I changed SQL schema and set DEFAULT '' to acl_name. After that, the upgrade did his job.

**Comment by WillyXJ** [comment]
Are you able to reproduce the first failure to find out why it occurred in the first place?

**Comment by internethering** [comment]
Yes:

1. Step: 1st run of upgradefmDNS_220():

1. do /** Run queries */: this runs the ALTER TABLE statements & db got changed
2. do /** Rework ACL table */: failed because of `#1364 - Field 'acl_name' doesn't have a default value`

2. Step: try to run database upgrade again, 2nd run of upgradefmDNS_220():

1. do /** Run queries */: database got changed at 1st run, so it failes now at this point. You should catch the 1060-error and ignore it ;)

**Comment by WillyXJ** [comment]
Confirmed bug when migrating the ACLs to the new schema.  I'd recommend rolling back to 2.1.5 until a fix is released as your ACLs likely did not carry over properly.

**Comment by WillyXJ** [comment]
This is fixed in v2.2.1.
